src/mlpack/bindings/cli/add_to_po.hpp

```diff
@@ -2,110 +2,117 @@
  * @file add_to_po.hpp
  * @author Ryan Curtin
  *
- * Utility functions to add options to boost::program_options based on their
+ * Utility functions to add options to CLI11 based on their
  * type.
  *
  * mlpack is free software; you may redistribute it and/or modify it under the
  * terms of the 3-clause BSD license.  You should have received a copy of the
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
-#ifndef MLPACK_BINDINGS_CLI_ADD_TO_PO_HPP
-#define MLPACK_BINDINGS_CLI_ADD_TO_PO_HPP
+#ifndef MLPACK_BINDINGS_CMD_ADD_TO_PO_HPP
+#define MLPACK_BINDINGS_CMD_ADD_TO_PO_HPP
 
 #include <mlpack/core/util/param_data.hpp>
-#include <boost/program_options.hpp>
 #include <mlpack/core/util/is_std_vector.hpp>
 #include "map_parameter_name.hpp"
 
+#include <CLI/CLI.hpp>
+
 namespace mlpack {
 namespace bindings {
-namespace cli {
+namespace cmd {
```

Comment:
I think we can still leave it as namespace `cli` since the CLI11 namespace is `CLI`; so there should be no collision.